lib/cylc/config.py

```diff
@@ -184,6 +184,14 @@ def __init__(self, suite, fpath, template_vars=[],
 
         graph_found = False
         for item, value in dependency_map.items():
+            if item == 'graph':
+                for line in value.split('\n'):
+                    if re.search(r'&&', line):
+                        linemsg = line.strip()
+                        raise SuiteConfigError(
+                            "Illegal double ampersand in '%s' at %s" 
```

Comment:
In retrospect, having seen where the code is, it's probably best to write "&&" rather than "double ampersand" here to disambiguate it from an item named "double ampersand".